ProjectSettings/TagManager.asset

```diff
@@ -13,7 +13,7 @@ TagManager:
   - UI
   - 
   - 
-  - 
+  - SpatialProcessingPlane
```

Comment:
Not sure we should be adding anything to the tag manager manually.

When people import this example, tags are generally not included.

Reply:
It'll probably be better to hard-code this in the example code.

Reply:
I didn't want to modify the code in SurfaceMeshesToPlanes.cs where it assigns dummy layer 31
![2018-05-17 12_43_53-mrtk - microsoft visual studio](https://user-images.githubusercontent.com/13754172/40200015-082a65ea-59d0-11e8-88cd-4ce9757dbdf4.png)

Solver Surface Magnetism script uses the layer through this drop down.
![2018-05-17 12_42_19-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40199932-cac9e52c-59cf-11e8-8286-3b8aa764d90f.png)

Reply:
It was just an example of how to implement a dialog to prompt devs to add the layer in their project.

Reply:
Sorry I just read #1443. In this case, I think we can use layer 31:Spatial Mapping and add instruction to use Configure > Apply Mixed Reality Project Settings.
Thank you!

![2018-05-17 12_49_47-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200329-dc8cc4a4-59d0-11e8-8f23-05e0d06bdbbe.png)

![2018-05-17 12_49_40-unity 2017 2 1p2 64bit - spatialprocessingsurfacemagnetism unity - mrtk - univ](https://user-images.githubusercontent.com/13754172/40200330-dca1a7fc-59d0-11e8-988c-9e0339ad3d71.png)